### PR TITLE
12.0 financial indicators

### PR DESCRIPTION
**Pull request by Danisan**
[ADD] financial indicator module. This module requests the UF, UTM, EUR and USD observed value each day from the former SBIF (now called "CMF.cl"). Requires an API Token to work.

### CODE REVIEW

File: l10n_cl_financial_indicators/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "version": "12.0.1.0.0",
+    "category": "Tools",
+    "development_status": "Production/Stable",
+    "maintainers": ["danisan"],
+    "license": "LGPL-3",
+    "author": "Blanco Martin & Asociados, Odoo Community Association (OCA)",
+    "website": "http://blancomartin.cl",
+    "depends": ["base"],
+    "data": [
+        "data/decimal.precision.xml",
```
Comment (dreispt) [inline l10n_cl_financial_indicators/__manifest__.py:12]: You should add decimal_precision dependency.

File: l10n_cl_financial_indicators/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+{
+    "name": "Chilean Financial Indicators",
```
Comment (dreispt) [inline l10n_cl_financial_indicators/__manifest__.py:2]: This seems rather specifically related to currencies, and not to generically financial /accounting related, correct?
If so I suggest to improve and narrow down the modules name and description.

Comment (Danisan) [reply l10n_cl_financial_indicators/__manifest__.py:2]: Daniel, what would be the best approach to the name? (in Spanish this is referenced as "indicadores financieros" that's why the name was that).

A proposal could be "l10n_cl_special_currency_rates" for example. What do you think?

Comment (Danisan) [reply l10n_cl_financial_indicators/__manifest__.py:2]: Even that, UF and UTM are not currencies; they are indexes or indicators but should be treated inside Odoo as currencies.

File: l10n_cl_financial_indicators/__manifest__.py (hunk 1 above)
Comment (Danisan) [reply l10n_cl_financial_indicators/__manifest__.py:12]: Hello Daniel.

Solved at f150b8

Do you need a rebase?

File: l10n_cl_financial_indicators/__manifest__.py (hunk 2 above)
Comment (Danisan) [reply l10n_cl_financial_indicators/__manifest__.py:2]: I will close this, since can see that @nelsonramirezs already change the module name.

Thnks!

**Pull request closed without merging**